data/block/blockV2.go

```diff
@@ -650,3 +650,19 @@ func (hv2 *HeaderV2) CheckFieldsForNil() error {
 
 	return nil
 }
+
+// GetPreviousAggregatedSignatureAndBitmap returns the previous aggregated signature and the previous pubkeys bitmap
+func (hv2 *HeaderV2) GetPreviousAggregatedSignatureAndBitmap() ([]byte, []byte) {
+	if hv2.Proof == nil {
+		return nil, nil
+	}
+	return hv2.Proof.PreviousAggregatedSignature, hv2.Proof.PreviousPubkeysBitmap
+}
+
+// SetPreviousAggregatedSignatureAndBitmap sets the previous aggregated signature and the previous pubkeys bitmap
+func (hv2 *HeaderV2) SetPreviousAggregatedSignatureAndBitmap(aggregatedSignature []byte, pubkeysBitmap []byte) {
+	hv2.Proof = &Proof{
+		PreviousPubkeysBitmap:       pubkeysBitmap,
```

Comment:
if the header field is called PreviousHeaderProof, then these could be PubKeysBitmap and AggregatedSignature